.github/workflows/bundle-size/package.json

```diff
@@ -7,11 +7,10 @@
     "build": "node build.js"
   },
   "dependencies": {
-    "@cloudscape-design/components": "*",
-    "@vitejs/plugin-react": "^4.0.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "typescript": "^5.1.3",
-    "vite": "^4.5.2"
+    "@vitejs/plugin-react": "^4",
+    "react": "^18",
+    "react-dom": "^18",
+    "typescript": "^5",
+    "vite": "^5"
```

Comment:
I also upgraded from Vite 4 to Vite 5

Reply:
Good doing an upgrade, but how about keeping the default npm behavior here? It adds full version number here.

I know it does not mean much because the real version is in the lock file, but having to manually format the file after installing a new module is annoying

Reply:
Hm, I find it more readable to only see the major version, since it more clearly communicates what we inted to install, e.g. Vite 5, React 18, etc. If you add another dependency with e.g. `npm install node-fetch` it does not modify the other entries, so there is no manual maintenance required

Reply:
You still will have to chase down inconsistencies on every code review. We already got similar story with `*` versions in some other cases, it did not scale super well.

Not a blocking comment anyway, just sharing some experience.